VideoCalls/Info.plist

```diff
@@ -27,6 +27,11 @@
 	</array>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSAllowsArbitraryLoads</key>
+		<true/>
+	</dict>
```

Comment:
I know you have used this for development :)
But we do not want to remove ATS from this project, so we should remove this code from this PR.

Reply:
Will do!